fuzzing/testdata/contracts/cheat_codes/vm/warp.sol

```diff
@@ -1,6 +1,6 @@
 // This test ensures that the block timestamp can be set with cheat codes
 interface CheatCodes {
-    function warp(uint64) external;
+    function warp(uint256) external;
```

Comment:
Question, should we leave compatibility for both uint256 and uint64 in case someone is already dependent on uint64, or do you think it doesn't matter? That's the only thing I thought worth mentioning here.

Reply:
Given that most people use Foundry, they will already have an interface setup to use with medusa and I doubt they will add the `warp(uint64)` function signature. So, we _can_, but I don't think it will serve any major value since most people will use HEVM's or Foundry's predefined interface.

Reply:
The primary thing we have to do is make sure we highlight this deviation in our documentation, which I have made note of.

Reply:
Awesome, thanks for the context :)